### ISSUE TITLE
RD3 : Execution panel doesn't refresh

### ISSUE THREAD

**Issue by Daryes** [rundeck/rundeck#4240]
**Describe the bug**
The Execution panel appearing after starting a job and following the execution doesn't update

**My Rundeck detail**

* Rundeck version: 3.0.8 (same problem with 3.0.7)
* install type: deb
* OS Name/version: ubuntu 18.04
* DB Type/version: H2 file

**To Reproduce**
1. start a job, with "following the execution" activated
2. wait for the job to finish
3. the job is already ended, but the status will still be on "Running", with the duration set to "+0.00:00 of ..." and the kill option visible in the upper right.
Yet, the job is already finished.
It'll correct itself if the page is refreshed on the browser.

**Expected behavior**
Previously, the Execution panel was updating the job state and duration, at least each second.

**Desktop (please complete the following information):**
 - OS: Windows 7
 - Browser Firefox 62.0.3

**Comment by gschueler** [comment]
is there any error in the JS console?

**Comment by Daryes** [comment]
There are : 
````
23:02:04.224 [Show/hide message details.] XHRGET https://rundeck.<domain>/execution/ajaxExecState/5235 [HTTP/1.1 200 OK 69ms]
23:02:04.262 Empty string passed to getElementById(). jquery-5393bf5931b3c4e7052edeec0d67327b.js:174:12
23:02:04.295 Error: Network Error
````

Also, it seems I've found the culprit : the UI keep looping on this call : 
```` 
XHRGET https://rundeck.<domain>/execution/ajaxExecState/5235 [HTTP/1.1 200 OK 69ms]
```` 
There's a response payload, with the job data. 
Problem is, even after the job is finished, the ajaxExecState keeps returning the `running` status for the job. Seems it's not updating with the real status of the job

**Comment by Daryes** [comment]
Fixed with 3.0.13, was due to the presence of a SSL terminated reverse proxy
